docs/DeveloperGuide.md

```diff
@@ -502,6 +542,75 @@ MSS:
 
    Use case ends.
 
+**Use case: UC0X - Calculate fee for a student of a particular month and year**
+
+MSS:
+
+1. User enters the `fee` commands with the appropriate inputs.
+2. TutorBuddy verifies that the inputs are valid.
+3. TutorBuddy verifies that the student profile exists.
+4. TutorBuddy shows the calculated fees to the user.
+
+   Use case ends.
+
+Extensions:
+
+* 2a. TutorBuddy detects an error in the entered command.
+    * 2a1. TutorBuddy displays error messages to the user.
+
+  Use case ends.
+
+* 3a. TutorBuddy detects that the user does not exist.
+    * 3a1. TutorBuddy displays an error message for unknown student profiles.
+
+  Use case ends.
+
+**UC0X - View 3 months monthly fee**
+
+MSS:
+
+1. User toggles to the `Home` tab.
+2. TutorBuddy shows the monthly fee that the user would have received for the past 3 months based on current sessions in the application.
+
+   Use case ends.
+
+**Use case: UC0X - Show help**
+
+MSS:
+
+1. User enters the `help` command.
+2. TutorBuddy displays a help window that contains a list of commands available on the application.
```

Comment:
Would adding "and a link to our user guide" be good here? Or is it too much information?

Reply:
Hmmm also can, I guess people can't say that having more information is a 'bug' ah, so I just put ah, seems safer.